doc/content/tutorials/efas/introduction/concluding_remarks.md

```diff
@@ -0,0 +1,54 @@
+
+!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/max_temp_vs_sim_time.png
+       style=width:100%;margin-left:auto;margin-right:auto;
+       id=max-temp-vs-sim-time
+       caption=Max Temperature vs Simulation Time.
+
+Regarding the contour plots at max temperature formed by ParaView, we observed patterns regarding temperature and electric potential. The graphite die wall thickness increase leads to a cooler temperature of the powder and die wall as shown by the ‘temperature’ contour plot, in [potential_temp_screenshot] for example. The temperature difference due to die wall thickness is visualized in the figures below by a sharp color shade difference between the 5 mm and 25 mm die thickness contour plots shown below. Also, this color change and thus gradual temperature decrease is seen progressively across all five die wall thickness examples. Especially if viewing the contour plots starting from the 5 mm die wall thickness temperature contour plot, then ending with the 25 mm die wall thickness temperature contour plot. Also, the center of the graphite plungers is generally hotter than the die wall and the powder casing, with an almost white color having an approximate value of 2.3e+03 K. 
+
+The electric potential on the toolset steadily decreases from the top of the toolset to the bottom of it, as observed in the contour plot below. This indicates that the direction of electric current is from top to bottom, and this current direction corresponds with how the DCS-5 machine is constructed. 
+
+!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/potential_temp_screenshot.png
+    style=width:80%;margin-left:auto;margin-right:auto;
+    id=potential_temp_screenshot
+    caption=Electrical Potential and Temperature Contour Plots.
+
+Electric field values slightly between different wall thickness, but noticeably changes on the graphite spacers. The electric field is highly concentrated in the center of these graphite spacers and seems to correspond with the higher temperature values in the same spot. This phenomenon is shown in [10mm_elec_field_and_temp], for example. However, the powder casing and die wall do not experience a similar spike in temperature or electric potential to the graphite spacers and may need to be considered when analyzing an EFAS toolset.
+
+!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/5mm_elec_field_and_temp.png
+    style=width:80%;margin-left:auto;margin-right:auto;
+    id=5mm_elec_field_and_temp
+    caption=Electric Field and Temperature Contour Plots for a 5 mm wall thickness at t = 954 seconds.
+
+!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/10mm_elec_field_and_temp.png
+    style=width:80%;margin-left:auto;margin-right:auto;
+    id=5mm_elec_field_and_temp
+    caption=Electric Field and Temperature Contour Plots for a 10 mm wall thickness at t = 960 seconds.
```

Comment:
Remove the `t =` in this caption and all others
```suggestion
    caption=Electric Field and Temperature Contour Plots for a 10 mm wall thickness at a simulation time of 960 seconds.
```

Reply:
Make this change for all the captions that have an equal sign to remove that equal sign